components/RemoveAccountComponent.js

```diff
@@ -0,0 +1,15 @@
+import SettingsPage from "../pageobjects/SettingsPage";
+
+export default class RemoveAccountComponent extends SettingsPage {
```

Comment:
extend base component but not particular page

components should not have access to properties and methods of page